### ISSUE TITLE
Clarification on aria-braillelabel SHOULD vs. MUST guidelines

### ISSUE THREAD

**Issue by rahimabdi** [w3c/aria#1995]
# Describe your concern

The editor's draft of ARIA 1.3 states the following:

> When using aria-braillelabel, authors SHOULD also ensure that:
> 
> 1. The element to which aria-braillelabel is applied has a valid accessible name.
> 2. The value of aria-braillelabel is not empty or does not contain only [whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) characters.
> 3. The value of aria-braillelabel does not contain any characters in [Unicode Braille Patterns](https://w3c.github.io/aria/#dfn-unicode-braille) or consists of only characters in [Unicode Braille Patterns](https://w3c.github.io/aria/#dfn-unicode-braille); the value does not only contain Braille Pattern dots-0.
> 4. The value of aria-braillelabel is not identical to the element's accessible name.

Should condition 1 (the element already has a valid accessible name) be a MUST rather than SHOULD? Denoting this condition as a SHOULD may imply that a valid accessible name is optional where `aria-braillelabel` is specified. If `aria-braillelabel` "...is meant to be used only if the accessible name cannot provide an adequate braille representation", does this mean that a valid accessible name MUST be provided if `aria-braillelabel` is specified?

# Link to the version of the specification or documentation you were looking at at.

Link to documentation: https://w3c.github.io/aria/#aria-braillelabel

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)? Yes.

**Comment by pkra** [comment]
@jnurthen @spectranaut quick ping to put this on the agenda again/finally.

**Comment by cookiecrook** [comment]
ARIA WG discussed and agreed to keep this as-is... Warnings allow permissiveness for unanticipated edge cases that an error would unnecessarily discourage... Some of the examples mentioned today were:

1. A braille "test" that purposefully overrode a speech label, as that would give away the answer to hearing SR users. (`aria-braillelabel` was requested by an education company for a similar purpose)... 
2. Similar teaching scenarios that compare different variants of braille, such as EBAE to UEB, or Nemeth vs 8-dot Computer Braille: As an example, contractions in UEB—such as "and" ("⠯")—can bridge a prefix like "pan-" in "pandemic". UEB is ⠏⠯⠑⠍⠊⠉ (6 cells representing 8 letters) and EBAE remains uncontracted ⠏⠁⠝⠙⠑⠍⠊⠉
